### PR TITLE
River delete playerpage

### PR DESCRIPTION
**Pull request by esump**
player page contents moved to song page

### CODE REVIEW

File: src/pages/song_page.tsx [hunk 1]
```diff
@@ -90,19 +157,133 @@ export const SongPage = (props: Props) => {
...
+        id='pause-button'
+        className='playPauseButton'
+        hidden={!isLoaded || !playing}
+        onClick={() => pause()}>
+        <AiOutlinePauseCircle className='playPauseIcon'/>
+      </button>
+      <div onClick={() => setDisplay(Display.Info)}>
+        <AiOutlineInfoCircle className='infoIcon'/>
+      </div>
+    </div>;
```
Comment (michael-hewitt) [inline src/pages/song_page.tsx:262]: Love all these functions.  Maybe a more descriptive name for `renderBottomDiv` might be something like `renderButtons` or perhaps you might have a better idea?

File: src/pages/song_page.tsx [hunk 2]
```diff
@@ -111,25 +292,17 @@ export const SongPage = (props: Props) => {
...
-  }
+    </div>
+  )}
 
   return (
     <div>
+      {renderInfoPage()}
       {renderPdf()}
       {renderPlayer()}
     </div>
```
Comment (michael-hewitt) [inline src/pages/song_page.tsx:308]: Gorgeous!

File: src/pages/song_page.tsx [hunk 3]
```diff
@@ -90,19 +157,133 @@ export const SongPage = (props: Props) => {
...
+    )
   }
 
   function renderPlayer() {
-    return <div key='player' style={{...(display === Display.PDF && {display: 'none'})}}>
+    return (
+    <div
+      key='player'
+      style={{...(display === Display.PDF && {display: 'none'})}}
+      hidden={display !== Display.SongSettings}>
```
Comment (michael-hewitt) [inline src/pages/song_page.tsx:286]: Love this display enum - makes the code very easy to understand.

**Review by michael-hewitt — APPROVED** [review]
I believe this is the right step.  Nice work!

File: src/pages/song_page.tsx (hunk 3 above)
Comment (esump) [reply src/pages/song_page.tsx:286]: It's certainly better than calling a function that makes state changes like I originally did in settings page. The enum on this page was actually here before me tho. I used it as a model for the settings page enum.

**Pull request merged**